api/namex/services/name_request/builders/name_analysis_builder.py

```diff
@@ -444,10 +444,12 @@ def get_most_similar_names(self, dict_highest_counter, matches, dist_substitutio
         list_desc = list(desc_synonym_dict.keys())
         if matches:
             syn_svc = self.synonym_service
+            nproc_svc = self.name_processing_service
             service = ProtectedNameAnalysisService()
             np_svc = service.name_processing_service
             wc_svc = service.word_classification_service
             token_svc = service.token_classifier_service
+            stand_alone_words = nproc_svc.get_stand_alone_words()
```

Comment:
There should be a separate function to check for dist desc dist pattern as well as dist dec desc pattern where they are in the same category for all names not embeded in the stand-alones functions..